api/SourceBufferList.json

```
@@ -15,17 +15,24 @@
               "prefix": "WebKit"
             }
           ],
-          "chrome_android": {
-            "version_added": "33"
-          },
+          "chrome_android": [
+            {
+              "version_added": "31"
+            },
+            {
+              "version_added": "25",
+              "version_removed": "31",
+              "prefix": "WebKit"
+            }
+          ],
           "edge": {
             "version_added": "12"
           },
           "firefox": {
             "version_added": "42"
           },
           "firefox_android": {
-            "version_added": false
+            "version_added": "42"
```

Matching most of the existing data. I'm not sure if it's right thought:

```suggestion
            "version_added": "41"
```

And can you update all of the subfeatures? `length` is now left false, which can't be right.